docs/pages/includes/database-access/gcp-public-address-note.mdx

```diff
@@ -0,0 +1,9 @@
+<Admonition
+  type="tip"
+  title="Tip"
+>
+  The Teleport Database service can connect to public or private addresses.
+  To use a private address the Teleport Database service must be able to connect
```

Comment:
Something feels off about this sentence - "must be able to connect being in the same zone"

Reply:
Please see update